pkg/services/rediscache/arm_template.go

```diff
@@ -1,112 +1,40 @@
-// +build experimental
-
 package rediscache
 
 // nolint: lll
 var armTemplateBytes = []byte(`
 {
-	"$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+	"$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
```

Comment:
Not clear on why this PR reverts this ARM template to use an older version of the management API for Redis...

Reply:
I used portal to generate the arm template and it had only this version and i checked few other modules and they had the same. So i thought it is the correct version.